base/lthooks.dtx

```diff
@@ -6183,7 +6185,9 @@
       { #2 { The~hook~is~empty } }
       {
         \@@_log_line:x { Code~chunks: }
-        \prop_if_empty:cTF { g_@@_#1_code_prop }
+        \bool_lazy_or:nnTF
+          { ! \prop_if_exist_p:c { g_@@_#1_code_prop } }
+          { \prop_if_empty_p:c { g_@@_#1_code_prop } }
```

Comment:
actually, it might be worth putting in a short comment why this is necessary, ie that a non-existent prop would return "empty" = true but that fail when mapping over it

Reply:
rubbish, would be empty = false ....